glance/files/mitaka/glance-api.conf.Debian

```diff
@@ -799,55 +799,101 @@ stores = {{ server.storage.engine }}
 # (dict value)
 #http_proxy_information =
 
+{%- if 'swift' in storage_engines %}
```

Comment:
You should add note about swift backend into readme.
Also metadata structure should be better, eg.:

```
glance:
  storage:
    swift:
      store:
        auth_insecure: false
        cacert: /etc/ssl/certs/ca-certificates.crt
```